pandas/core/window/ewm.py

```diff
@@ -275,8 +256,10 @@ def mean(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args, **kwargs
-            Arguments and keyword arguments to be passed into func.
+        *args
+            Arguments to be passed into func.
+        **kwargs
+            Keyword arguments to be passed into func.
```

Comment:
Better revert this please.

Reply:
Any reason to not resolve this and mark as resolved? I think your proposed change is incorrect, the original is more accurate.

Reply:
No idea why I missed these. I'll get them right away.

Reply:
Actually, it looks like I just forgot to mark this as resolved. It should be fixed in the last commit.

Reply:
I think it's not reverted.

Reply:
Can you revert please

Reply:
For some reason it's showing up as reverted on my machine, but not here. I'll just do the edits on Github.